WalletWasabi.Gui/WalletWasabi.Gui.csproj

```diff
@@ -64,6 +64,8 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Avalonia" Version="0.8.2" />
```

Comment:
Two new packages? Why? What do they contain?

Reply:
This overrides Avalonia to 0.8.2 (since they are binary compatible)

We actually only need Avalonia.Native, since the only code change was in Avalonia.Native.

Avalonia.Native is the OSX backend.

In 0.9 release it will be a requirement for the xamlil compiler that we reference Avalonia.Desktop only from the csproj.

Reply:
Could we keep only AvalonStudio.Shell? I don't like the direction this is going.